ckanext/switzerland/logic.py

```diff
@@ -260,7 +260,8 @@ def ogdch_cleanup_harvestjobs(context, data_dict):
                 model.Session.execute(sql)
 
                 # reindex after deletions
-                tk.get_action('harvest_sources_reindex')(context, {})
+                tk.get_action('harvest_source_reindex')(
+                    context, {'id': source.id})
```

Comment:
I would add to the documentation, that the command won't run a whole reindex and advise to consider to do that manually afterwards in the documentation. other than that, this looks good to me.

Reply:
Done: https://projects.liip.ch/display/OGDCH/Harvestjob+Cleanup